app/routes/articles/components/ArticleDetail.tsx

```diff
@@ -77,13 +78,7 @@ const ArticleDetail = ({
             {authors?.map((e, i) => {
               return (
                 <span key={e.username}>
-                  <Link
-                    to={`/users/${e.username}`}
-                    className={styles.overviewAuthor}
```

Comment:
Why remove the class? It is still used to add dots between the names, if I'm not mistaken.

Reply:
There's no `.overviewAuthor` class in `ArticleDetail.css`, so it didn't do anything.

Reply:
We had a [similar thing with `.eventwarning`](https://github.com/webkom/lego-webapp/pull/3133) a while back. It will sometimes find the correct css class, even though it does not link to the correct css file. So I believe it is still used, but only sometimes lol. It's most likely a bug, so please fix it instead. We'd want the `.overviewAuthor` styling anyways.

Reply:
![Screenshot 2023-04-17 at 17 34 49](https://user-images.githubusercontent.com/8343002/232537085-ca519b4e-bca5-43f2-8d11-b1ef44c7ba56.png)
wayy to many dots when I add it.

Reply:
Is it because of multiple authors? Very weird behaviour